README.md

```diff
@@ -27,6 +27,17 @@ $ impl 's *Source' golang.org/x/oauth2.TokenSource
 func (s *Source) Token() (*oauth2.Token, error) {
     panic("not implemented")
 }
+
+# You can request that the declaring file be updated in-place by passing -u
+$ impl -u 'f *File' io.ReadWriteCloser
```

Comment:
Why not `-w`, since that's what common Go tools like `gofmt` and `goimports` use?

Reply:
Good suggestion. I'll switch to that flag.